sources/packages/api/src/utilities/system-configurations-constants.ts

```diff
@@ -59,3 +59,10 @@ export const ECERT_PART_TIME_FILE_CODE = "PBC.EDU.PTCERTS.D";
  */
 export const MSFAA_FULL_TIME_FILE_CODE = "PBC.EDU.MSFA.SENT.";
 export const MSFAA_PART_TIME_FILE_CODE = "PBC.EDU.MSFA.SENT.PT.";
+
+/**
+ * These constants are used to specify the filename code
+ * created for Full-Time/ Part-Time feedback file.
+ */
+export const ECERT_FULL_TIME_FEEDBACK_FILE_CODE = "EDU.PBC.CERTSFB.*\\.DAT";
```

Comment:
`FILE_CODE` does not seems accurate. It seems that this should be something like `FILE_NAME`. I can see other constants in the same file following the same, so please rename all or leave them as it is.

Reply:
I thought of it more like a filename Code, if you see the comments, I have explained. Eventually as this is a variable used in the regex of the filename, i thought to go with code. Didnt think a lot on the variable name, it came eventually